modular_zubbers/code/modules/reagents/chemistry/reagents/other.dm

```diff
@@ -0,0 +1,9 @@
+
+//Splash-based recipe to turn improvised gauze into sterile gauze.
+/datum/reagent/space_cleaner/sterilizine/expose_obj(obj/O, reac_volume)
+	. = ..()
+	if(istype(O, /obj/item/stack/medical/gauze/improvised))
+		var/obj/item/stack/medical/gauze/G = O
+		reac_volume = min((reac_volume/10), G.amount)
+		new /obj/item/stack/medical/gauze(get_turf(G), reac_volume)
```

Comment:
I have no idea how to interpret the reac_volume arg, and I can't find what this line would call.

Reply:
I'll explain this as best I can. The proc itself calls the initial definition of reac_amount, which is the amount of reagent splashed on the object. This equation then divides the initial amount by 10 and rounds down to redefine the variable, which is then used to spawn one piece of gauze for however many full multiples of 10u was sprayed, so long as you have enough stacks of improvised gauze to make into gauze.